tests/test_modeling_nystromformer.py

```diff
@@ -274,6 +274,25 @@ def test_model_from_pretrained(self):
             model = NystromformerModel.from_pretrained(model_name)
             self.assertIsNotNone(model)
 
+    @unittest.skip(reason="Head masking not implemented")
+    def test_head_pruning(self):
+        pass
+
+    @unittest.skip(reason="Head masking not implemented")
+    def test_head_pruning_integration(self):
+        pass
+
+    @unittest.skip(reason="Head masking not implemented")
+    def test_head_pruning_save_load_from_config_init(self):
+        pass
+
+    @unittest.skip(reason="Head masking not implemented")
+    def test_head_pruning_save_load_from_pretrained(self):
+        pass
+
+    @unittest.skip(reason="Head masking not implemented")
+    def test_headmasking(self):
+        pass
```

Comment:
These tests can actually also be skipped by setting `test_headmasking=False` and `test_pruning=False` to the `ModelTester`. Example here: https://github.com/huggingface/transformers/blob/e68c3756fea7c811d02b8470539ae17ec3ec0e71/tests/test_modeling_wav2vec2.py#L398